### PR TITLE
Export the GMFs in csv format

### PR DESCRIPTION
**Pull request by micheles**
Closes https://bugs.launchpad.net/oq-engine/+bug/1409620. Notice that the file names have been changed to avoid collisions in the case of oversampling. Moreover, now the TRT numbering starts from zero for consistency with oq-lite.
The tests are running here: https://ci.openquake.org/job/zdevel_oq-engine/1029

### CODE REVIEW

File: openquake/engine/export/hazard.py [hunk 1]
```diff
@@ -100,55 +101,31 @@ def _get_result_export_dest(calc_id, target, result, file_ext='xml'):
             ltr = result.lt_realization
```
Comment (ptormene) [inline openquake/engine/export/hazard.py]: This block looks exactly the same as the case of `output_type == 'gmf'`

**Comment by ptormene** [comment]
Please fix the 5 broken tests

**Comment by micheles** [comment]
I have fixed the tests

**Comment by ptormene** [comment]
LGTM

**Pull request merged**